nutanix/services/clustersv2/data_source_nutanix_cluster_entities_v2.go

```diff
@@ -100,10 +100,16 @@ func DatasourceNutanixClusterEntitiesV2Read(ctx context.Context, d *schema.Resou
 		return diag.Errorf("error while fetching cluster entities : %v", err)
 	}
 
-	getResp := resp.Data.GetValue().([]import1.Cluster)
+	if resp.Data == nil {
```

Comment:
did we checked the issue in v3 modules as well? If not better to check them as well.